src/pages/Downloads.js

```diff
@@ -1,22 +1,121 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import { Page } from '../components'
 import { Text } from '@aragon/ui'
 import styled from 'styled-components'
 import Section from '../components/Page/Section'
+import figmaSVG from './assets/figma.svg'
+import SketchSVG from './assets/sketch.svg'
+import FramerSVG from './assets/framer.svg'
+import DownloadIconSVG from './assets/downloadicon.svg'
 
 const Downloads = () => (
   <Page path="/downloads">
     <Section>
-      <Container />
+      <Container>
+        <PageWrapper>
+          <H1>Downloads</H1>
+          <Content>
+            <Description><P>Download some goodies to start designing your decentralized apps. This is an ever growing collection of components for Sketch and Figma, and interactive components for Framer X.</P><P>If you would like to contribute to designing new components, please get in touch! The best way is to open an issue on <A href={"https://github.com/lorikeetui/design/issues"}>https://github.com/lorikeetui/design/issues</A></P></Description>
+            <DownloadWrapper>
+              <DownloadLink src={SketchSVG} text={"Download for Sketch"} href={"#"}/>
+              <DownloadLink src={figmaSVG} text={"Download for Figma"} href={"#"}/>
+              <DownloadLink src={FramerSVG} text={"Download for Framer X"} href={"#"}/>
+            </DownloadWrapper>
+          </Content>
+        </PageWrapper>
+      </Container>
     </Section>
   </Page>
 )
 
+const DownloadLink = (props) =>
+  <LinkWrapper>
+    <IconContainer><Icon src={props.src} /></IconContainer>
+    <DownLink href={props.href}><InsideLinkWarpper><LinkText>{props.text}</LinkText><DownloadIcon src={DownloadIconSVG}/></InsideLinkWarpper></DownLink>
+  </LinkWrapper>
+
+const H1 = styled.h1`
+  font-size: 46px;
+  color: #fff;
+`
+
+const Content = styled.div`
+  display: flex;
+  margin-top: 40px;
+`
+
+const P = styled.p`
```

Comment:
Maybe this styled component could desappear and we can put this padding inside Description component